infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationValidation.kt

```diff
@@ -347,7 +346,7 @@ private fun validateExcessTracksThroughJoint(
     validatingTrack: LocationTrack?,
 ): PublishValidationError? {
     val excesses = tracksThroughJoint.filter { (joint, tracks) ->
-        joint != connectivityType.sharedJoint && tracks.count(LocationTrack::isDraft) > 1
```

Comment:
:meat_on_bone: 
Tämä varmaankin oli näin koska ennen kontekstia julkaisujoukko oli vähän hähmäisempi käsite, mutta nyt kontekstista pitäisi tulla juuri ne raiteet jotka lopputuloksessa on -> voidaan vain validoida että kaikki on hyvin.